charms/worker/charmcraft.yaml

```diff
@@ -64,6 +64,12 @@ platforms:
 
 config:
   options:
+    bootstrap-certificates:
+      type: string
+      default: "self-signed"
+      description: |
+        The type of certificates to use in Canonical Kubernetes. This cannot be
+        changed after deployment. Allowed values are "self-signed" and "external".
```

Comment:
Can this config be shared over the `cluster` relation application databag rather than charm config?  Is there a use-case where either:
* the control-plane self-signs, but the worker doesn't
* the worker self-signs, the worker doesn't

Reply:
Great idea!